src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java

```diff
@@ -68,6 +78,94 @@ private AdminSearchPage getAdminSearchPage(String instructorId) {
         return loginAdminToPage(browser, commentsPageUrl, AdminSearchPage.class);
     }
     
+    private boolean isPageTitleCorrect() {
+        return searchPage.getPageTitle().equals("Admin Search");
+    }
+    
+    private boolean isSearchPanelVisible() {
+        return searchPage.isElementPresent(By.id("filterQuery"))
+            && searchPage.isElementPresent(By.id("searchButton"));
```

Comment:
If you are checking for the visibility of the search panel, you might want to use isElementVisible instead. Or If you are checking for the presence of the search panel, perhaps you could rename the method to `isSearchPanelPresent()` instead